### PR TITLE
Translation to Bosnian language

### PR DESCRIPTION
**Pull request by demirkaric**

### CODE REVIEW

**Comment by kenjis** [comment]
@demirkaric Thank you for sending the PR!

It seems `bs` is a new locale, so unit testing is missing.
See https://github.com/codeigniter4/translations/blob/develop/CONTRIBUTING.md#extending-unit-tests

**Comment by kenjis** [comment]
Check failed.

> 1) Translations\Tests\BosnianTranslationTest::testAllIncludedLanguageKeysAreTranslated with data set "bs" ('bs')
Failed asserting that the translated language keys "Migrations.batch", "Migrations.namespace" in "bs" locale differ from the original keys in the main repository.

**Review by kenjis — APPROVED** [review]

**Comment by demirkaric** [comment]
@kenjis Just one thing about keys Migrations.batch and Migrations.namespace, can we keep original values from English for those two keys? It's hard to find words that will properly fit those two keys in the Bosnian language.

File: tests/Language/AbstractTranslationTestCase.php [hunk 1]
```diff
@@ -66,6 +66,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         VietnameseTranslationTest::class         => 'vi',
         SimplifiedChineseTranslationTest::class  => 'zh-CN',
         TraditionalChineseTranslationTest::class => 'zh-TW',
+        BosnianTranslationTest::class            => 'bs',
```
Comment (paulbalandan) [inline tests/Language/AbstractTranslationTestCase.php:69]: Please put this after the `ar` above.

**Comment by paulbalandan** [comment]
Please add `bs` to the table in README

**Comment by paulbalandan** [comment]
> @kenjis Just one thing about keys Migrations.batch and Migrations.namespace, can we keep original values from English for those two keys? It's hard to find words that will properly fit those two keys in the Bosnian language.

@demirkaric If that's the case, you still need to write the keys in the **Migrations.php** file. Then, for tests to pass, add this property to your `BosnianTranslationTest`:
```php
final class BosnianTranslationTest extends AbstractTranslationTestCase
{
    protected array $excludedLocaleKeyTranslations = [
        'Migrations.batch',
        'Migrations.namespace',
    ];
}
```

File: tests/Language/AbstractTranslationTestCase.php (hunk 1 above)
Comment (demirkaric) [reply tests/Language/AbstractTranslationTestCase.php]: Fixed it. Thank you.

**Comment by demirkaric** [comment]
> > @kenjis Just one thing about keys Migrations.batch and Migrations.namespace, can we keep original values from English for those two keys? It's hard to find words that will properly fit those two keys in the Bosnian language.
> 
> @demirkaric If that's the case, you still need to write the keys in the **Migrations.php** file. Then, for tests to pass, add this property to your `BosnianTranslationTest`:
> 
> ```
> final class BosnianTranslationTest extends AbstractTranslationTestCase
> {
>     protected array $excludedLocaleKeyTranslations = [
>         'Migrations.batch',
>         'Migrations.namespace',
>     ];
> }
> ```

Thank you. I did it that way.

**Comment by paulbalandan** [comment]
@demirkaric I do not see you added `bs` to the README, and code style is failing.

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
Thank you, @demirkaric

**Pull request merged**